### DISCUSSION TITLE
Uppercase Snippets

### DISCUSSION THREAD

**Discussion by wt-gao** [krvajal/vscode-fortran-support#858] [Q&A - deprecated see Fortran Discourse] [answered]
When I prefer uppercase Fortran code, the autocompletion code snippets and some keywords are lowercase, which is very embarrassing. I think this plugin should automatically switch when I select a Fortran code in uppercase or lowercase form.

**Comment by gnikit** [comment, marked as answer]
Read the instruction and check your settings, the extension already does that.
There are some limitations e.g. with preset snippets, which is not the same as autocompletions.
